src/Types.ts

```diff
@@ -0,0 +1,34 @@
+export type Node = {
+  x: number;
+  y: number;
+  value: number;
+  isExit: Boolean;
+};
+
+export type Direction = "N" | "S" | "E" | "W";
+
+export type Edge = {
+  from: string;
+  to: string;
+  direction: Direction;
+};
+
+export type Graph = {
+  nodes: {
+    [coordinates: string]: Node;
+  };
+  edges: Array<Edge>;
+};
+
+export type Board = Array<Array<number>>;
+
+export type Deriv = {
```

Comment:
Why use a truncated word and not a complete word?

Reply:
has been renamed